upkie/cpp/actuation/RobotSimulator.h

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2024 Inria
+
+#pragma once
+
+#ifndef ROBOT_SIMULATOR_CLIENT_API_H
+#define ROBOT_SIMULATOR_CLIENT_API_H
+
+#include <Eigen/Dense>
+
+#include "Bullet3Common/b3Logging.h"
+#include "RobotSimulator/b3RobotSimulatorClientAPI.h"
+#include "SharedMemory/PhysicsClientC_API.h"
+#include "SharedMemory/b3RobotSimulatorClientAPI_InternalData.h"
+
+//! Structure which contains the new link properties
+struct RobotSimulatorChangeDynamicsArgs {
+  //! The mass of the robot link
```

Comment:
SI units please :smiley: (whenever possible, it is mental type checking for us as well)